src/Server.ts

```diff
@@ -1,21 +1,40 @@
 import * as express from "express";
+import * as bodyParser from "body-parser";
+import notFoundRoute from "./libs/routes/notFoundRoute";
+import errorHandler from "./libs/routes/errorHandler";
```

Comment:
import from 'libs/routes' not from file itself. create index file in routes and export these function from there

Reply:
fixed import as per feedback also lint the code